page_queue.go

```diff
@@ -87,6 +110,15 @@ func (ui *Ui) createQueuePage() *QueuePage {
 	return &queuePage
 }
 
+func (q *QueuePage) changeSelection(row, column int) {
+	q.songInfo.Clear()
+	if row >= len(q.queueData.playerQueue) || row < 0 || column < 0 {
+		return
+	}
+	currentSong := q.queueData.playerQueue[row]
+	_ = q.songInfoTemplate.Execute(q.songInfo, currentSong)
```

Comment:
Can the last line not fail?

Reply:
No.

Technically, yes, but the template is hard-coded in the source;  it's not loaded from a file or is dynamic in any way, and the data used is typed, so if it works during testing, it _can't_ fail during runtime.